res/css/structures/_SpaceRoomView.pcss

```diff
@@ -27,12 +27,10 @@ $SpaceRoomViewInnerWidth: 428px;
     font-weight: $font-semi-bold;
```

Comment:
here, in fact

Reply:
Right, addressed with https://github.com/matrix-org/matrix-react-sdk/pull/10554/commits/9e30d1b96ca8d65835a8d43cfa67d82fe24bdd07.